syntax = "proto3";

message BlurArgs {
  int32 kernel_size = 1;
  float sigma = 2;
}

message ResizeArgs {
  int32 width = 1;
  int32 height = 2;
  bool min = 3;
  bool preserve_aspect = 4;
  string interpolation = 5;
}
